locations/container/src/main/java/org/apache/brooklyn/container/location/kubernetes/KubernetesLocation.java

```diff
@@ -347,8 +386,8 @@ public Boolean call() {
                 if (result.isEmpty()) {
                     return false;
                 }
-                List<HasMetadata> check = client.resource(result.get(0)).inNamespace(result.get(0).getMetadata().getNamespace()).get();
-                if (result.size() > 1 || check.size() != 1 || check.get(0).getMetadata() == null) {
+                HasMetadata check = client.resource(result.get(0)).inNamespace(result.get(0).getMetadata().getNamespace()).get();
```

Comment:
Is that normal the signature changed from `List<HasMetadata>` to `HasMetadata` ?

Reply:
think so, but need to double-check as I don't recall the details